domains/arturomauricio.is-cool.dev.json

```diff
@@ -8,7 +8,7 @@
     },
 
     "record": {
-        "CNAME": "arturomauricio"
+        "CNAME": "https://arturomauriciodev.github.io/arturomauriciodev/"
```

Comment:
Invalid CNAME record. CNAME cannot have other characters than dots and letters/numbers.